npm/ng-packs/packages/feature-management/src/lib/components/feature-management/feature-management.component.html

```diff
@@ -64,16 +64,20 @@ <h4>{{ selectedGroupDisplayName }}</h4>
                   <div class="form-group mb-0">
                     <label [htmlFor]="feature.name">{{ feature.displayName }}</label>
                     <select class="form-control" [id]="feature.name" [(ngModel)]="feature.value">
-                      <option [ngValue]=""></option>
-                      <option
-                        *ngFor="
-                          let prop of feature.valueType.properties | keyvalue;
-                          trackBy: track.by('key')
-                        "
-                        [ngValue]="prop.value"
-                      >
-                        {{ prop.key }}</option
-                      >
+                      <ng-container *ngIf="feature.valueType.itemSource?.items?.length">
```

Comment:
If the items don't exist, shouldn't we hide the select altogether instead of an empty select box?